schemas/org.w3/PerformanceNavigationTiming/jsonschema/1-0-0

```diff
@@ -0,0 +1,231 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for page navigation performance, based on the PerformanceNavigationTiming interface (see https://w3c.github.io/navigation-timing/)",
```

Comment:
Do we have a convention for denoting when something is intended to be used primarily as an Entity vs Event (I know there's technically no such distinction, but it can help establish some context when you're browsing existing schemas)? I usually just mention one or the other in the top description but if we have something more formal that's good to know.

Reply:
Yeap adding this in.